### DISCUSSION TITLE
Une activité annexe

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#2433] [Nouvelles idées]
L'idée consiste à introduire la pêche dans DraftBot.

On aurait un inventaire pêche qui affiche nos poissons et avec l'arrivée des boutons, on peut juste faire une 3eme page de l'inventaire actuel

FORMAT
L'action principale serait sous forme de mini-évent, disponible seulement sur les trajets impliquant le Lac Mirage. Grossièrement on fait une pause au bord du lac et le mini-évent consiste à choisir son spot de pêche du style "au bord de la rive", "sur le ponton à votre gauche", "sur une barque".

PRINCIPE 
Vendre les poissons péchés avec une commande /sellfish (lol). Il y aurait plusieurs variété et donc rareté de poisson, valant plus ou moins cher. Après avoir fait /sellfish, on choisis l'espèce de poisson à vendre puis la quantité, 1/5/10/all.

IDÉES BONUS
- En plus des poissons, la pêche offre une très faible chance de pêcher quelques gemmes (entre 1 et 4?), un item random ou encore un trésor (à définir, peut être une caisse contenant 2k💰). 
- Au moment de l'obtention du poisson (donc juste après avoir réagi au mini-évent et choisi son spot), on pourrais avoir un message "souhaitez vous placer votre prise dans votre inventaire ou le donner à votre pet ?". La 2eme option donne des pts de dressage comme un /petfeed (seule faille, les pets herbivores...)

**Comment by BastLast** [comment]
ptdr et si tu pousse le pecheur tu élimine la concurrence et ya plus de poissons pendant une petite période après

**Reply by KirIcare** [reply]
J'ai pensé faire un lien avec ce mec ou même avec l'item canne à pêche mais flemme xD

**Comment by KirIcare** [comment]
Ah et j'ai dis Lac Mirage mais si ça marche bien on peut l'étendre à tt les trajets contenant de l'eau (je veux dire les même qui permettent l'accès à l'île mystérieuse)

**Comment by Voisin1234** [comment]
J'aime beaucoup l'idée!!! Après avec la V5, je suis pas sûre qu'elle va être mis tout de suite si celle-ci est acceptée. ou alors on va peut-être te dire de faire de la pub pour qu'on soit au moins 1500 joueurs par semaine( oui, c'est une petite pique gratuite 🤭😋 ) C'est vrai qu'il y a plein de petits choses comme ça qu'on pourrait mettre en place mais ton idée à mon goût est top et bien détaillé. Pour une fois j'ai tout compris 👍

**Reply by BastLast** [reply]
jpense on sera plus sur 2K5 joueurs dans le tw là

**Reply by Voisin1234** [reply]
😂
